R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -371,13 +371,13 @@ test_that("Collect DataFrame with complex types", {
   expect_equal(bob$height, 176.5)
 })
 
-test_that("read.json()/jsonFile() on a local file returns a DataFrame", {
```

Comment:
How about renaming this test case to "read/write json files", where read.json(), jsonFile(), read.df(), write.json(), write.df() are tested to read/write json files. This test case can be merged with the one "read.df() from json file"